From c55810ad0de0b5351ed5578cd82dc7f8824e9aea Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 18 Feb 2020 13:45:10 +0200
Subject: [PATCH] [SW-8229] goya2: fix efuse spi controller bit

Efuse bit for enabling spi controller is at bit 17, not 20

Change-Id: I15275da881bc30b99e1489d4895c3296cd348572
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/common/efuse.h b/common/efuse.h
index 552a6b7..267b700 100644
--- a/common/efuse.h
+++ b/common/efuse.h
@@ -17,7 +17,7 @@
 #define EFUSE_SYSCFG_OFFSET			0x3A
 #define EFUSE_SECURITY_FW_VERSION_OFFSET	0x3B
 
-#define EFUSE_SYSCFG_SPI_CONTROLLER		BIT(20)
+#define EFUSE_SYSCFG_SPI_CONTROLLER		BIT(17)
 #define EFUSE_SYSCFG_DISABLE_DEBUG_IF_MASK	BIT(12)
 #define EFUSE_SYSCFG_FW_VERSION_LOCATION	13
 #define EFUSE_SYSCFG_FW_VERSION_LOCATION_MASK	0x3
